clients/drcachesim/tests/syscall_insertion.templatex

```diff
@@ -0,0 +1,8 @@
+Basic counts tool results:
+Total counts:
+      [1-9][0-9][0-9][0-9][0-9][0-9] total \(fetched\) instructions
+        5971 total unique \(fetched\) instructions
```

Comment:
If we hardcoded this number, should we hardcode the others as well as they don't vary?  Or you're just trying to have fewer values to change if we update the data file?  Ditto below.

Reply:
Same initial rationale as the original clients/drcachesim/tests/switch_insertion.templatex.

Hardcoded the userspace and kernel instr count for all three template files. Shouldn't be too much extra effort to replace the two additional numbers.